src/main/java/com/google/sps/servlets/KeywordServlet.java

```diff
@@ -26,79 +22,21 @@
 @WebServlet("/keyword")
 public class KeywordServlet extends HttpServlet {
 
-  private final List<String> tweets = new ArrayList<>();
-  private final TreeMap<Entity, Float> terms = new TreeMap<>((term1, term2) -> {
-    // Sort the Entity keys by their salience scores
-    if (term1.getSalience() > term2.getSalience()) {
-      return 1;
-    } else if (term1.getSalience() < term2.getSalience()) {
-      return -1;
-    } else {
-      return 0;
-    }
-  });
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String key = request.getParameter("k"); // TODO: Use key to access datastore.
-    for (String tweet : tweets) {
-      AnalyzeEntitiesResponse entityResponse = analyzeEntity(tweet);
-      List<Entity> entities = entityResponse.getEntitiesList();
-      for (Entity entity : entities) {
-        if (entity.getSalience() == 0) {
-          continue;
-        }
-       
-        // We are only looking at (at most) the 10 most salient keywords. 
-        if (terms.size() < 10) {
-          if (terms.containsKey(entity)) {  // Keep the max salience score if the term already exists in the map.
-            terms.put(entity, Math.max(entity.getSalience(), terms.get(entity)));
-            continue;
-          }
-          terms.put(entity, entity.getSalience());
-        } else if (!terms.containsKey(entity) && entity.getSalience() > terms.get(terms.pollFirstEntry())) {
-          terms.remove(terms.pollFirstEntry());
-          terms.put(entity, entity.getSalience());
-        } else {}
-      }
-    }
-    
-    // TODO: Retrieve tweets from datastore.
-    List<String> keywords = new ArrayList<>();
-    for (Entity entity : terms.keySet()) {
-      keywords.add(entity.getName());
-    }
+    Set<String> keywords = Keywords.getKeywords();
```

Comment:
You should be probably passing the datstore key to this method, if not the same set of keywords would be returned to any client.